src/spec/sync/utils/product.spec.coffee

```diff
@@ -1249,6 +1249,63 @@ describe 'ProductUtils', ->
       update = @utils.actionsMapAttributes delta, originalProduct, newProduct
       expect(update.length).toBe(0)
 
+    it 'should create update action if attribute value item is removed', ->
+      newProduct =
+        masterVariant:
+          sku: 'TEST MASTER VARIANT'
+          attributes: [
+            {
+              'name': 'test_attribute',
+              'value': [
+                'a', 'b'
+              ]
+            }
+          ]
+
+      originalProduct =
+        masterVariant:
+          sku: 'TEST MASTER VARIANT'
+          attributes: [
+            {
+              name: 'test_attribute',
+              value: [
+                'a', 'b', 'c'
+              ]
+            }
+          ]
+
+      delta = @utils.diff originalProduct, newProduct
+      update = @utils.actionsMapAttributes delta, originalProduct, newProduct
+      expect(update.length).toBe(1)
+
+    it 'should create update action if attribute value item is added', ->
+      newProduct =
+        masterVariant:
+          sku: 'TEST MASTER VARIANT'
+          attributes: [
+            {
+              'name': 'test_attribute',
+              'value': [
+                'a', 'b', 'c'
+              ]
+            }
+          ]
+
+      originalProduct =
+        masterVariant:
+          sku: 'TEST MASTER VARIANT'
+          attributes: [
+            {
+              name: 'test_attribute',
+              value: [
+                'a', 'b'
+              ]
+            }
+          ]
+
+      delta = @utils.diff originalProduct, newProduct
+      update = @utils.actionsMapAttributes delta, originalProduct, newProduct
```

Comment:
Can we check the value of the update action, to be sure that it's the right update action 😉